asset/asset.go

```diff
@@ -1077,6 +1077,11 @@ func (a *Asset) HasGenesisWitnessForGroup() bool {
 	return *witness.PrevID == ZeroPrevID
 }
 
+// IsGenesisAsset returns true if an asset is a genesis asset.
+func (a *Asset) IsGenesisAsset() bool {
```

Comment:
non-blocking nit: there are at least three places where we have this exact condition that we could shorten with this new method.